v0/destinations/yahoo_dsp/util.js

```diff
@@ -0,0 +1,283 @@
+const qs = require("qs");
+const sha256 = require("sha256");
+const { generateJWTToken } = require("../../../util/jwtTokenGenerator");
+const {  httpPOST } = require("../../../adapters/network");
+const { CustomError, isDefinedAndNotNullAndNotEmpty } = require("../../util");
+
+const { ACCESS_TOKEN_CACHE_TTL } = require("./config.js");
+const Cache = require("../../util/cache");
+
+const ACCESS_TOKEN_CACHE = new Cache(ACCESS_TOKEN_CACHE_TTL);
+
+const getUnixTimestamp = () => {
+  return Math.floor(Date.now() / 1000);
+};
+
+/**
+ * There are four location types which are stored in the poiLocationType error. These are ["chains", "woeids", "gids", "categories"].
+ * The ones to be added in includes Object will be taken from audienceType with include as Prefix. The ones to be added in excludes
+ * Object will be taken from audienceType with exclude as Prefix.
+ */
+const poiLocationType = [
+  "includeChains",
+  "includeWoeids",
+  "includeCategories",
+  "includeGids",
+  "excludeChains",
+  "excludeWoeids",
+  "excludeCategories",
+  "excludeGids"
+];
+let listType;
```

Comment:
These should be context specific declarations i.e within functions

Reply:
updated